src/utils/helpers.js

```diff
@@ -29,3 +29,59 @@ export const createIdsSelector = createSelectorCreator(memoizeResult);
 
 // Use this selector when you want a shallow comparison of the arguments and you don't need to memoize the result
 export const createShallowSelector = createSelectorCreator(defaultMemoize, shallowEqual);
+
+// isMinimumServerVersion will return true if currentVersion is equal to higher or than the
+// the provided minimum version. A non-equal major version will ignore minor and dot
+// versions, and a non-equal minor version will ignore dot version.
+// currentVersion is a string, e.g '4.6.0'
+// minMajorVersion, minMinorVersion, minDotVersion are integers
+export function isMinimumServerVersion(currentVersion, minMajorVersion = 0, minMinorVersion = 0, minDotVersion = 0) {
```

Comment:
we couldn't move getting the current server version into this function? I feel like it would be nice if the function got the server version itself, instead of it being passed in.

Reply:
IMO utility functions shouldn't be pulling data but should have it all passed to them. Easier for testing and for use in cases where the store/whatever might not have the values you need set

Reply:
Essentially it completely decouples them from anything but the caller

Reply:
okay. If that is the reason, I am good with that. I just think it isn't very DRY having `const serverVersion = Client4.getServerVersion();` so many times; especially since the server shouldn't change between checks. 
Also, is Client4.getServerVersion() memoized?

Reply:
No it's not memoized but it's just a string and there's no calculations going on there so I don't think memoizing it would be useful

Reply:
To get around having multiple `const serverVersion = Client4.getServerVersion()` we could have a helper function in the actions folder that wraps the utility but I don't think it's warranted just yet

Reply:
@jwilander okay, I thought it may be round tripping to the server each time to check. 👍

Reply:
Oh, yeah it's just a string stored as a property on the Client4 class

Reply:
We could treat this like a selector depending on where we're using it. Like you just pass in state and a version requirement, and it automatically returns whether or not it meets that requirement. 0/5 though